src/App.js

```diff
@@ -78,19 +150,44 @@ const App = () => {
               exact
               path="/workouts"
               component={() => (
-                <YourWorkouts selectedExercises={selectedExercises} />
+                <YourWorkouts
+                  selectedExercises={selectedExercises}
+                  removeEx={removeEx}
+                  setExercisePage={setExercisePage}
+                  setFormDialog={setFormDialog}
+                  formDialog={formDialog}
+                  workoutName={workoutName}
+                  setWorkoutName={setWorkoutName}
+                />
               )}
             />
             <Route
               exact
-              path="/results"
+              path="/:results"
```

Comment:
What does adding a : to the route do?

Reply:
This makes the routing dynamic. So instead of each muscle group saying localhost:3000/results for each page, it would say localhost:3000/neck.

Think we should put a comment here as well?

Reply:
Yeah for sure! Also maybe could you send me a link to resource that can teach me how to do it? Seems great!

Reply:
Definitely! I watched this and adapted what he did with ours. It took me a minute to figure it out, but I eventually got it.
Video: [https://www.youtube.com/watch?v=4NpGzBEySvI](url)